src/app/signup/signup.component.ts

```diff
@@ -24,72 +26,112 @@ export class SignupComponent implements OnInit {
                    phone: new FormControl(''),
                     });
 
 
                      
   	            }
-  
+
+
+   
   ngOnInit() {
+    this.loadExternalScript('https://www.google.com/recaptcha/api.js?render=6LdoiLwZAAAAANJ-MV-ZORWzs8IwU1IjDPJcXnvn');
   
   	
   }
+  public loadExternalScript(url: string) {
+  const body = <HTMLDivElement> document.body;
+  const script = document.createElement('script');
+  script.innerHTML = '';
+  script.src = url;
+  script.async = true;
+  script.defer = true;
+  body.appendChild(script);
+}
+
+  
+
   /*
  * This function is used for signup
- * @params(email:string,password:string,user id:string,phone number:string)
- * As  no live api url found we are setting token static value  
+ * @params(email:string,password:string,user name:string,phone number:string)   
  */
-
   signupFormSubmit(){
-      this.isSubmitted = true;
-       const response = grecaptcha.getResponse();
-      console.log("forms"+response);
-      if (response.length === 0) {
-          console.log("not");
-          this.errormsg = 'Recaptcha not verified. Please try again!';
-           this.snackbar.open('Recaptcha not verified. Please try again!','OK',{
-                verticalPosition: 'top',
-                horizontalPosition:'right'
-              });
-          return;
-        }
+      this.isSubmitted = true;         
+      
+      
       if(this.signupForm.valid)
       {
-      let saveData = this.signupForm.value;   
-       
+       let saveData = this.signupForm.value;   
+          grecaptcha.ready(() => {
+              grecaptcha.execute('6LdoiLwZAAAAANJ-MV-ZORWzs8IwU1IjDPJcXnvn', { action: 'cta_signup' }).then((token) => {
+                   console.log(token);
+                  this.authService.captchaVerify(token).pipe(first()).subscribe(res => {                     
+
+                       if(res['score'] >=0.5){                       
+                            
+                             this.authService.SignUp(saveData).pipe(first()).subscribe(signupres => {
+                    
+                    if(signupres.status.status_code == 200)
+                      {
+                       console.log("ok"); 
+                       this.snackbar.open('Registered successfully','OK',{
+                          verticalPosition: 'top',
+                          horizontalPosition:'right'
+                        });
+                      location.href = 'tax-prepare-profile';
+                       
+                      }
+                      else if(signupres['status'].status_code == 400){
+                         this.snackbar.open(signupres['status']['message'],'OK',{
+                          verticalPosition: 'top',
+                          horizontalPosition:'right'
+                        });
+                      }
+                       else{                      
+                       confirm('Sorry, an error occurred. Please email support@digitaltaxusa.com');
```

Comment:
@Bithika-Redapple You are using this same error message multiple times. Create a Constant out of it and have one point of reference.